### PR TITLE
demo update

### PR DESCRIPTION
**Pull request by NEGRAH-S**
## Project

Sunbird Observe - Revamp the observe sink 

### Changes

 Updated (new pages, weekly update, etc.)

### Notes

N/A

### CODE REVIEW

**Comment by GeekGawd** [comment]
You haven't the folder name Frontend and Deep Linking kindly change it

**Comment by NEGRAH-S** [comment]
@GeekGawd hey made changes. kindly check and merge.

**Comment by GeekGawd** [comment]
@NEGRAH-S kindly remove the new folder and keep the folder structure as 2023/Sunbird/project-name, also update the slug to keep it the same name the as the folder names.

**Comment by NEGRAH-S** [comment]
@GeekGawd can we connect on discord for once

**Comment by NEGRAH-S** [comment]
@GeekGawd u can check did the final changes

File: docs/2023/Sunbird Obsrv/revampTheObsrvSink/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: Introduction
+slug: /2023/Sunbird Obsrv/revampTheObsrvSink
+---
+
+
+# Author
+Negrah
+
+# Contact Info
+ [Email](negrahsingh32@gmail.com)
```
Comment (photon0205) [inline docs/2023/Sunbird Obsrv/revampTheObsrvSink/index.md:12]: change this to `[Email](mailto:negrahsingh32@gmail.com)`

File: docs/2023/Sunbird Obsrv/revampTheObsrvSink/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+[Linked In](NegrahS) 
+[GitHub](NEGRAH-S)
```
Comment (photon0205) [inline docs/2023/Sunbird Obsrv/revampTheObsrvSink/index.md:14]: add full link with https for both github and linkedin

**Review by photon0205 — CHANGES_REQUESTED** [review]

**Review by photon0205 — APPROVED** [review]

**Review by NEGRAH-S — COMMENTED** [review]
thnx for approval @photon0205 @GeekGawd

File: docs/2023/Sunbird Obsrv/revampTheObsrvSink/index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+
+# Author
+Negrah
+
+# Contact Info
+[Email](mailto:negrahsingh32@gmail.com)
+[Linked In](https://www.linkedin.com/in/negrahs23/) 
+[GitHub](https://github.com/NEGRAH-S)
```
Comment (claddyk) [inline docs/2023/Sunbird Obsrv/revampTheObsrvSink/index.md:14]:
```suggestion
- [Email](mailto:negrahsingh32@gmail.com)
- [Linked In](https://www.linkedin.com/in/negrahs23/) 
- [GitHub](https://github.com/NEGRAH-S) 
```

**Review by claddyk — APPROVED** [review]

**Pull request merged**